string group_name
string[] pose_names
PoseConfigMsg[] pose_configs
uint32 total_points